### PR TITLE
[venice-common] Use interpolation format for logging for helix.

### PR DESCRIPTION
**Pull request by xuhao417347761**
## Summary, imperative, start upper case, don't end with a period

This rb mainly contains the logging clean ups for helix related code in venice-common module .

## How was this PR tested?

Internal CI (yellow)
## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixStatusMessageChannel.java [hunk 1]
```diff
@@ -171,7 +171,7 @@ public void sendToStorageNodes(String clusterName, StatusMessage message, String
       }
     }
     if (isSuccessful) {
-      LOGGER.info(numMsgSent + " messages have been send and processed. Message: {}.", message.toString());
+      LOGGER.info("{} messages have been send and processed. Message: {}.", numMsgSent, message.toString());
```
Comment (sushantmane) [inline internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixStatusMessageChannel.java:174]: remove toString

**Review by sushantmane — APPROVED** [review]
lgtm

File: internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixStatusMessageChannel.java (hunk 1 above)
Comment (xuhao417347761) [reply internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixStatusMessageChannel.java:174]: Sure, let me check all this toString usages in this change.

Comment (sushantmane) [reply internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixStatusMessageChannel.java:174]: Thank you

**Review by sushantmane — APPROVED** [review]
Thanks, Hao!

**Pull request merged**